images/base/files/scripts/bootstrap_pipeline.py

```diff
@@ -11,7 +12,14 @@
 
 
 def run_pipeline(config):
-    if not os.path.exists("pipeline/pipeline.py"):
+    if os.environ["HEXA_PIPELINE_NOTEBOOK"]:
+        notebook_path = f'./workspace/{os.environ["HEXA_PIPELINE_NOTEBOOK"]}'
```

Comment:
`Path("/home/jovyan/workspace") / os.environ["HEXA_PIPELINE_NOTEBOOK"]`
Don't forget to check if the file exists as well.

I'm not a fan of using an environment variable to check if it is a notebook or a pipeline that needs to be ran.

Reply:
What I have in mind and I'm trying to put on paper is to use arguments passed to the command line instead of relying exclusively on envvars. In the meantime, I think we can live with the  `HEXA_NOTEBOOK_PATH`.